Flutter/flutter_gallery_app/lib/src/presentation/button/screen/button_page.dart

```diff
@@ -0,0 +1,221 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/button/bloc/button_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/button/bloc/button_event.dart';
+import 'package:flutter_gallery_app/src/presentation/button/bloc/button_state.dart';
+import 'package:flutter_gallery_app/src/presentation/button/widgets/custom_border_button.dart';
+import 'package:flutter_gallery_app/src/presentation/button/widgets/option_button.dart';
+
+@RoutePage()
+class ButtonPage extends StatelessWidget {
+  const ButtonPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<ButtonBloc>(
+      create: (context) => ButtonBloc(),
+      child: const ButtonView(),
+    );
+  }
+}
+
+class ButtonView extends StatefulWidget {
```

Comment:
You should separate the 'ButtonView' class into a separate file. And then change name "ButtonView" to "ButtonWidget"

Reply:
I have resolved the problem in new commit